### ISSUE TITLE
Flax Editor crashes when printing error in Linux

### ISSUE THREAD

**Issue by Crawcik** [FlaxEngine/FlaxEngine#732]
**Issue description:**
When editor must print an error it throws an exception/assertion:
```
Assertion failed!
File: /home/crawcik/Repozytoria/FlaxEngine/Source/ThirdParty/fmt/format.h
Line: 1696
```

**Steps to reproduce:**
1. Use Linux
2. Do some syntax error in code

**Flax version:**
master & 1.3

[Log.txt](https://github.com/FlaxEngine/FlaxEngine/files/9100373/Log.txt)

**Comment by Crawcik** [comment]
Problem has been fixed!

**Comment by mafiesto4** [comment]
Fix commit: 2cd7967ec494d5d5be9bf8d798538a3f530bdb11
